doc/README.md

```diff
@@ -12,16 +12,8 @@ functionalities that you might need in a game. For instance; input, images, spri
 animations, collision detection, and a component system that we call Flame Component System (FCS for
 short).
 
-We also provide stand-alone packages that extend the Flame functionality:
-
-- [flame_audio](https://pub.dev/packages/flame_audio) Which provides audio capabilities using the
-  `audioplayers` package.
-- [flame_forge2d](https://pub.dev/packages/flame_forge2d) Which provides physics capabilities using
-  our own `Box2D` port called `Forge2D`.
-- [flame_tiled](https://pub.dev/packages/flame_tiled) Which provides integration with the
-  [tiled](https://pub.dev/packages/tiled) package.
-- [flame_svg](https://pub.dev/packages/flame_svg) Which provides integration with
-  [flutter_svg](https://pub.dev/packages/flutter_svg).
+We also provide stand-alone packages that extend the Flame functionality which can be found in the
+[Bridge Packages](#bridge_packages/bridge_packages) section.
```

Comment:
Seems like it doesn't like the link:
> doc/README.md:16:1 MD051/link-fragments Link fragments should be valid [Context: "[Bridge Packages](#bridge_packages/bridge_packages)"]

Did you try locally?

Reply:
I did yes!

Reply:
But lint failed, so I changed, tested and it still workds